_sass/_blog.scss

```diff
@@ -0,0 +1,179 @@
+.blog-sidebar {
+  background-color: $navy-drk;
+  border-radius: 2px;
+  height: 100%;
+  margin: {
+    top: 35px;
+    bottom: 48px;
+    right: 60px;
+  }
+}
+
+.recent-post-header {
+  font-size: 20px;
+  padding-left: 25px;
+}
+
+h3.recent-post-header {
+  margin: {
+    top: 10px;
+    bottom: 13px;
+  }
+}
+
+.recent-posts {
+  background-color: lighten($navy-drk, 3%);
+  padding: {
+    top: 5px;
+    bottom: 5px;
+  }
+  h2 {
+    padding-left: 25px;
+    font-size: 10px;
+    a {
+      color: $green;
+    }
+  }
+}
+
+.post,
+.index-list {
+  width: 100%;
+}
+
+.index-list {
+  display: flex;
+  flex-direction: column;
+  h1 {
+    margin-bottom: 0px;
+    margin-top: 22px;
+  }
+  p.meta {
+    margin-top: 0px;
+  }
+  .post {
+    margin-bottom: 30px;
+  }
+}
+
+.blog-posts,
+.post-wrapper {
+  display: flex;
+  flex-direction: row;
+}
+
+.post h1 a {
+  color: white;
+  font-size: 36px;
+  &:hover {
+    color: $green;
+    text-decoration: none;
+    border-bottom: 2px solid $green;
+  }
+}
+
+.post p.meta {
+  color: #8FA5BC;
```

Comment:
Please use only color variables. If this is a new color, create a new variable for it.